SharedSources/MediaLibraryService.swift

```diff
@@ -288,10 +288,28 @@ extension MediaLibraryService {
 
     func fetchMedia(with mrl: URL?) -> VLCMLMedia? {
         guard let mrl = mrl  else {
+            assertionFailure("no mrl")
```

Comment:
I believe it could be better to have a better string for the assertion see the other assertions on in the service.

Example: `assertionFailure("MediaLibraryService: Medialibrary already initialized.")`